rose/client/main.py

```diff
@@ -74,6 +74,14 @@ def main():
         d = {}
         exec f in d, d
 
-    reactor.connectTCP(d['server_address'], config.game_port,
+    if len(sys.argv) == 3:
+        server_address = sys.argv[2]
+    else:
+        server_address = d['server_address']
```

Comment:
This will ignore the user specified server_address in the driver module. I think we should either fail or warn about this. Or maybe remove the server address from the driver file and make the server address required in the client command line, same as the admin command line.